README.md

```diff
@@ -38,6 +38,20 @@ If your install of swift is located at `/swift` and you wish to install XCTest i
 ./build_script.py --swiftc="/swift/usr/bin/swiftc" --build-dir="/tmp/XCTest_build" --swift-build-dir="/swift/usr" --library-install-path="/swift/usr/lib/swift/linux" --module-install-path="/swift/usr/lib/swift/linux/x86_64"
 ```
 
+To run the tests, pass the `--test` option in combination with options to install XCTest in your active version of Swift:
+
+```sh
+./build_script.py --swiftc="/swift/usr/bin/swiftc" --build-dir="/tmp/XCTest_build" --swift-build-dir="/swift/usr" --library-install-path="/swift/usr/lib/swift/linux" --module-install-path="/swift/usr/lib/swift/linux/x86_64" --test
+```
+
+To add additional tests to XCTest:
+
+1. (Linux & OS X) Add a directory with the name of your test to the `Tests/Fixtures/Sources/` directory. For example, if your test is named "TwoFailingTestCases", make a directory named `Tests/Fixtures/Sources/TwoFailingTestCases`.
+1. (Linux & OS X) Add a `main.swift` file to the directory you added above. The file should contain comments, beginning with `// `. The test runner will verify that the output of `main.swift` matches your comments exactly. This is all you need to do to run tests on Linux--you may run the `build_script.py` example above to confirm the tests pass.
+1. (OS X) If you want your tests to run on OS X, open the `XCTest.xcodeproj` Xcode project and duplicate the `SingleFailingTestCase` target. Rename the target to match the name of your test. For example, if your test is named "TwoFailingTestCases", name the target `TwoFailingTestCases` as well.
```

Comment:
`Tests/test.py` works by running each executable in the `Tests/Fixtures/Products/` directory. Those need to be built **before** the script is run.

The `SingleFailingTestCase` Xcode target compiles `Tests/Fixtures/Sources/SingleFailingTestCase/main.swift` into an executable and installs it in the `Products` directory. It also ensures that `SwiftXCTest.framework` is linked properly.

So in order to run the test **on OS X**, adding a new Xcode target (steps 3 and 4) is necessary, because it is responsible for putting an executable in the `Products` directory. Adding the new target to the aggregate target (step 5) is necessary to eliminate the need to build the newly added target manually before running `Tests/test.py`.

Also, your comment reminded me: because the executables are compiled and placed in the `Products` directory every time the tests are run (whether on OS X or Linux), the contents of that directory should be added to the `.gitignore`. I'll amend that change to commit b52c686.